scipy/stats/_multivariate.py

```diff
@@ -960,6 +964,36 @@ def rvs(self, mean=None, rowcov=1, colcov=1, size=1, random_state=None):
             out = out.reshape(mean.shape)
         return out
 
+    def equivalent_multivariate_normal(self, mean=None, rowcov=1, colcov=1,
+                                       random_state=None):
```

Comment:
What is the point of this as a public method other than inviting usage with very wrong big-O characteristics? How about adding the transformation of the mean and cov as a recipe to the documentation (two lines of code + a paragraph of text)